cadence/src/sinks/core.rs

```diff
@@ -77,6 +139,10 @@ pub trait MetricSink {
     fn flush(&self) -> io::Result<()> {
         Ok(())
     }
+
```

Comment:
Can you add some docs here? In particular mention that not all sinks have stats.